cpp/ql/lib/experimental/semmle/code/cpp/semantic/SemanticExprSpecific.qll

```diff
@@ -286,19 +286,7 @@ module SemanticExprConfig {
     )
   }
 
-  class Bound instanceof IRBound::Bound {
-    string toString() { result = super.toString() }
-
-    final Location getLocation() { result = super.getLocation() }
-  }
-
-  private class ValueNumberBound extends Bound {
-    IRBound::ValueNumberBound bound;
-
-    ValueNumberBound() { bound = this }
-
-    override string toString() { result = bound.toString() }
-  }
+  class Bound = IRBound::Bound;
```

Comment:
This change looks like it's more than just "using instanceof in more places", no?

Reply:
> It's in experimental, so I think the API change is OK.

That's true. However, we _hope_ that this library won't be confined to experimental for that much longer 🤞.

> (The only visible API change is that the exported Bound class has all the predicates from IRBound::Bound).

Yeah, it's the increased public API surfaces that I'm most afraid of. What do you say to this @rdmarsh2?

Reply:
I see.  
I went for the minimal change instead.